lib/src/stream_subscription_transformer.dart

```diff
@@ -76,4 +99,7 @@ class _TransformedSubscription<T> extends DelegatingStreamSubscription<T> {
     if (_cancelMemoizer.hasRun) return;
     _handleResume(_inner);
   }
+
+  Future/*<E>*/ asFuture/*<E>*/([/*=E*/ futureValue]) =>
```

Comment:
I'd like to hold off on this until 1.22 stable is released, to preserve flexibility in version solving.